sql_generators/active_users/__init__.py

```diff
@@ -83,6 +90,81 @@ def generate(target_project, output_dir, use_cloud_function):
                     app_name=browser.name,
                 )
             )
+        # create checks_sql
+        if browser.name == "firefox_desktop":
+            checks_sql = desktop_checks_template.render(
+                project_id=target_project,
+                app_value=browser.value,
+                app_name=browser.name,
+            )
+        elif browser.name == "fenix":
+            checks_sql = fenix_checks_template.render(
+                project_id=target_project,
+                app_value=browser.value,
+                app_name=browser.name,
+            )
+        elif browser.name == "firefox_ios":
```

Comment:
I think there is a way to not require these multiple "if" and making it easy to add/remove sources in the future. We could declare a dictionary at the beginning of the file similar to:
{
"firefox_ios": ['moz-fx-data-shared-prod.org_mozilla_ios_firefox_live.baseline_v1', 'moz-fx-data-shared-prod.org_mozilla_ios_firefoxbeta_live.baseline_v1', 'moz-fx-data-shared-prod.org_mozilla_ios_fennec_live.baseline_v1'], "focus_ios": ['moz-fx-data-shared-prod.org_mozilla_ios_focus_live.baseline_v1'],
"klar_ios": ['moz-fx-data-shared-prod.org_mozilla_ios_klar_live.baseline_v1'], "focus_android": ['moz-fx-data-shared-prod.org_mozilla_focus_live.baseline_v1', 'moz-fx-data-shared-prod.org_mozilla_focus_beta_live.baseline_v1', 'moz-fx-data-shared-prod.org_mozilla_focus_nightly_live.baseline_v1']
}
and use it to compose the "channels". The mobile template template would be rendered only once.
It can also be part of a later PR! Your call.

Reply:
just when the CI is nearly done with checking .. but no you are right good point let me quickly do that